src/main/scala/Main.scala

```diff
@@ -34,7 +35,8 @@ object Main extends App {
     }
 
     val tag = "FNB"
-    val maxTokens = 50
+    val maxTokens = 30
+    val minTokens = 10
```

Comment:
Adding `minTokens` to just to ensure the sentence is not too short for training.